core/src/components/picker-column/picker-column.tsx

```diff
@@ -352,23 +365,7 @@ export class PickerColumn implements ComponentInterface {
            */
           this.canExitInputMode = true;
 
-          const dataIndex = activeElement.getAttribute('data-index');
-
-          /**
-           * If no value it is
-           * possible we hit one of the
-           * empty padding columns.
-           */
-          if (dataIndex === null) {
-            return;
-          }
-
-          const index = parseInt(dataIndex, 10);
-          const selectedItem = this.items[index];
-
-          if (selectedItem.value !== this.value) {
-            this.setValue(selectedItem.value);
-          }
```

Comment:
Before, we were dealing with `button` elements so we had to encode the button index so we could then find the associated item value. We don't need to do that with the option component since we can just look at the `value` property directly.